README.md

```diff
@@ -193,124 +200,167 @@ Other acceptable location examples:
+This maintains a consistent and straightforward user experience, allowing you to reuse logic and parameters across all tools.
 
-To select multiple locations in a single request you can use a comma. And of course you can combine filters and multiple locations at the same time.
+So, for example, when you run a test from "Germany" using the CLI or Slack app, our API's magic parser processes the location.
 
-- `from amazon+france,ovh+france --limit 2` will return a probe hosted with AWS in France and a probe hosted with OVH in France
+> [!Note]
+> Developers who want stricter and more predictable control over the selected probes and user input can use the individual location parameters [when making an API call](https://www.jsdelivr.com/docs/api.globalping.io#post-/v1/measurements), which expect you to provide each city, country, and tag in a standardized way.
 
-You can use this functionality to also define your own "world".
+The magic field supports a wide range of parameters and location combinations, including countries, continents, cities, US states, regions (Western Europe), ASNs, ISP names, eyeball or data center tags, and cloud region names (us-east-2).
 
-- `from germany, greece, uk, usa, canada, japan, china, south africa --limit 20` will return probes from all of the above countries in roughly equal proportions to fit the limit of 20. 
+### Re-select probes ♻️
+You can also provide the "magic" field with a measurement ID to have the API select the same probes used in a previous measurement. 
 
-The limit parameter is global and applies to the location as a whole, not allowing the API to return more than the limit. If you need to define custom limits per location you need to use our API directly instead of the magic field.
+For example:
+- `from WZIAtMx4LLhzit02`
 
+You can get the measurement ID in every Globalping measurement result bei using the `--share` or `--reuse` flags.
 
-### System tags (eyeball networks and more)
+This feature is particularly useful, for example, if you want to test different API endpoints with the same probes to better compare data. It is also great for troubleshooting, as you can use it to retest certain "problematic" probes or run continuous tests with the same probe and stitch the results together.
 
-Many probes are going to be tagged by our system. At the moment this includes:
+### Understand the "world" location 🌍
 
-- Cloud regions for Google Cloud and AWS. e.g. `aws-eu-west-1` and `gcp-us-south1` They follow each provider's naming scheme and are prefixed by their name.
-- Eyeball and datacenter networks. e.g. `eyeball-network` and `datacenter-network` Unlike datacenter hosted probes, eyeball network are probes hosted with ISP providers that offer internet access to regular people and small businesses. 
+The `world` location uses a pseudo-random algorithm to select probes while [maintaining certain proportions](https://github.com/jsdelivr/globalping/blob/master/config/default.cjs#L47) per continent.
```

Comment:
Maybe lets keep the "is special" part? To make it clear that this is the only dynamic location we have, everything else is based on static data